### PR TITLE
[4.0] Codestyle fixes for tests dir

### PR DESCRIPTION
**Pull request by 810**
Codestyle fixes.

### Testing Instructions

None

### Expected result

No issues

### CODE REVIEW

File: tests/Unit/Libraries/Cms/Access/RuleTest.php [hunk 1]
```diff
@@ -157,6 +156,11 @@ public function testAllow()
 		$this->assertFalse($rule->allow([-42, 2]));
 	}
 
+	/**
+	 *
+	 * @return void
```
Comment (mbabker) [inline tests/Unit/Libraries/Cms/Access/RuleTest.php:161]: IMO it needs to be a PHPCS rule that PHPUnit test methods don't require the "standard" doc block.  There's no real reason for it.  When you're using a data provider, it's good to document the `@param` values as usual, but there's no reason to force `@return void` on every test, nor is there a real reason to document when the test was added with a `@since` tag.

(Note the doc blocks are generally still useful when building abstract test cases, or documenting data providers, but forcing the Joomla code style on test methods is just a little much; part of the reason the doc blocks in general are in such poor shape is because the code style rules make them compulsory and people fill them in just to check the block without making them useful)

File: tests/Unit/Libraries/Cms/Console/Loader/WritableContainerLoaderTest.php [hunk 2]
```diff
@@ -16,11 +15,15 @@
...
 /**
  * Test class for Joomla\CMS\Console\Loader\WritableContainerLoader.
+ *
+ * @since   4.0.0
  */
 class WritableContainerLoaderTest extends UnitTestCase
 {
 	/**
-	 * @var  ContainerInterface|\PHPUnit_Framework_MockObject_MockObject
+	 * @var  ContainerInterface
```
Comment (mbabker) [inline tests/Unit/Libraries/Cms/Console/Loader/WritableContainerLoaderTest.php]: Incorrect change.  The `createMock()` method returns a `PHPUnit\Framework\MockObject\MockObject` instance (namespaced as of PHPUnit 8).  Documenting the type as both the `MockObject` and the mocked `ContainerInterface` helps with IDE resolution as it will provide hints for both object types (otherwise the `$this->container->exists()` call later in this class will end up with the IDE complaining the method doesn't exist on the object).

File: tests/Unit/Libraries/Cms/Access/RuleTest.php (hunk 1 above)
Comment (810) [reply tests/Unit/Libraries/Cms/Access/RuleTest.php:161]: We should teaching the basic joomla codestyle. If its not showed correctly even on tests systems, then the other devs may think why i should do codestyling at all.

Comment (mbabker) [reply tests/Unit/Libraries/Cms/Access/RuleTest.php:161]: This is enforcement of a coding style for the sake of checking a block.  A coding style should be practical and teaching good practices, not just "you need to do this for the sake of ensuring the CI suite passes".  Look at the number of doc blocks you have added which have absolutely zero value but pass the CI suite.

Comment (mbabker) [reply tests/Unit/Libraries/Cms/Access/RuleTest.php:161]: If you're going to add doc blocks to everything in the tests directory, they should be good doc blocks and not the minimum required to make the CI suite pass.  Adding doc blocks for the sake of it is quite frankly a waste of your time, if you're going to spend time on this then at least make it worthwhile.

Comment (810) [reply tests/Unit/Libraries/Cms/Access/RuleTest.php:161]: Ok i have done the Codeception dir. For now im stopping fixing it, so this can be merged. Later i will redo the other dirs.

**Comment by wilsonge** [comment]
Thanks!
